### PR TITLE
Prep for bonsai asset

### PR DESCRIPTION
**Pull request by jspaleta**
#### General

- [X] Update Changelog following the conventions laid out on [Our CHANGELOG Guidelines ](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [X] Update README with any necessary configuration snippets

- [X] RuboCop passes

#### Purpose
- Enable bonsai asset
- Remove unsupport ruby versions

#### Known Compatibility Issues
test-kitchen tests are disabled until fix for kitchen base image is found.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ This CHANGELOG follows the format listed at [Our CHANGELOG Guidelines ](https://
 Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- Update minimum required ruby version to 2.3. Drop unsupported ruby versions.
```
Comment (majormoses) [inline CHANGELOG.md]: missing a header here

**Comment by majormoses** [comment]
@jspaleta looks like you need to rebase

File: CHANGELOG.md (hunk 1 above)
Comment (jspaleta) [reply CHANGELOG.md]: fixed

**Comment by jspaleta** [comment]
rebase and updated changelog to reflect.
Ready for merge and minor release

**Pull request merged**